spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerOperations.java

```diff
@@ -192,9 +192,9 @@
 	/**
 	 * Update an object in storage.
 	 * @param object the object to update.
-	 * @param includeColumns the columns to update. If null or an empty Optional is given, then
-	 * all columns are used. Note that an Optional occupied by an empty Set means that no columns
-	 * will be used.
+	 * @param includeColumns the columns to update. If null or an empty Optional is given,
+	 * then all columns are used. Note that an Optional occupied by an empty Set means
+	 * that no columns will be used.
 	 */
 	void update(Object object, Optional<Set<String>> includeColumns);
```

Comment:
Generally `Optional` should not be used as a method parameter. See [discussion on SO](https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments).
Just allow it to be `null`.

Reply:
made new issue 
https://github.com/spring-cloud/spring-cloud-gcp/issues/1057